lib/OpenQA/WebAPI/Command/gru/run.pm

```diff
@@ -24,7 +24,7 @@ has usage       => sub { shift->extract_usage };
 sub run {
     my ($self, @args) = @_;
 
-    getopt \@args, 'o|oneshot' => \my $oneshot, 'reset-locks' => \my $reset_locks;
+    getopt \@args, ['pass_through'], 'o|oneshot' => \my $oneshot, 'reset-locks' => \my $reset_locks;
```

Comment:
I don't understand what this does. The commit says "Allow passing CLI options to minion" but nothing else is being added there. That's the only change in the commit besides the copyright 🤔

Reply:
Looks a bit weird, but is very sensible here. It's an option of the underlying param parsing module that leaves unknown flags in `@args` to be processed somewhere else. https://docs.mojolicious.org/Mojo/Util#getopt

Reply:
I'm actually not sure why it's missing, should have been there already.

Reply:
The commit message means that the change allows to *pass* options through "this layer of abstraction". So additional `@args` which go in are passed to the run function of the base class which then handles them (whatever they are). So you can e.g. specify `--jobs 10` and these args are passed to the Minion worker with no further handling from our side.

Reply:
Thank you for clarifying! Confusion resolved. 👍🏾